sdk/eventhub/Azure.Messaging.EventHubs/src/Azure.Messaging.EventHubs.csproj

```diff
@@ -12,24 +12,21 @@
     <Folder Include="Properties\" />
   </ItemGroup>
 
-  <ItemGroup><!-- This package will be removed when v5.3.0 is released for GA -->
+  <ItemGroup>
+    <!-- This package will be removed when v5.3.0 is released for GA -->
     <PackageReference Include="Microsoft.Azure.Amqp" />
     <PackageReference Include="Microsoft.Bcl.AsyncInterfaces" />
     <PackageReference Include="System.Diagnostics.DiagnosticSource" />
     <PackageReference Include="System.Reflection.TypeExtensions" />
     <PackageReference Include="System.Threading.Channels" />
     <PackageReference Include="System.Threading.Tasks.Extensions" />
+    <!--This can be removed once this dependency is added to Core in the next release.-->
+    <PackageReference Include="System.Memory.Data" />
   </ItemGroup>
 
-  <!-- Import the Azure.Core reference -->
+  <!-- Import the Azure.Core reference - Uncomment before next GA release -->
```

Comment:
I think we can remove the "Uncomment...." part, as this is no longer commented out.

Reply:
Yep, just noticed that. Also removed the misplaced comment about removing the below reference after GA which ended up above the AMQP lib 😆